eo-phi-normalizer/app/Main.hs

```diff
@@ -77,6 +79,8 @@ cliTransformPhiParser = do
   json <- jsonSwitch
   outputFile <- outputFileOption
   single <- switch (long "single" <> short 's' <> help "Output a single expression.")
+  maxDepth <- option auto (long "max-depth" <> value 10 <> help "Maximum depth of rules application. Defaults to 10.")
```

Comment:
I suggest to add a `metavar` like `NUMBER` so that it's not just `ARG` in the help.